bundles/org.openhab.binding.omnilink/src/main/java/org/openhab/binding/omnilink/internal/OmnilinkBindingConstants.java

```diff
@@ -104,7 +104,7 @@
     public static final String CHANNEL_ROOM_SCENE_D = "scene_d";
     public static final String CHANNEL_ROOM_STATE = "state";
 
-    public static final String CHANNEL_SYSTEMDATE = "sysdate";
+    public static final String CHANNEL_SYSTEMDATE = "system_date";
```

Comment:
This is a breaking change for anyone having already linked items to the old channel name. Is it needed?

Reply:
Not necessarily needed, but it would bring the naming of this channel in line with the style used on the other channels in the binding. I can omit the change if you think it would be better.

Reply:
@jlaur Would you prefer to revert this change or move forward. It is not necessary just brings the style in line with the rest of the code.

Reply:
@ecdye - three options as I see it. Up to you how much effort you want to put into it, I'm fine with any of them:
1. Leave it as is to not introduce a breaking change. It may not be pretty, but binding doesn't seem to conform to channel naming convention anyway.
2. Rename it and introduce a breaking change. This must be mentioned in release notes, so another PR for add a line to https://github.com/openhab/openhab-distro/blob/main/distributions/openhab/src/main/resources/bin/update.lst would be needed.
3. Rename it and provide backwards compatibility in code. See for example #11668.

Reply:
@jlaur I decided to go with option one for them moment, I would rather get this merged and save that change for when I have the time to do it properly.